src/SourceBuild/content/repo-projects/arcade.proj

```diff
@@ -3,6 +3,13 @@
   <PropertyGroup>
     <!-- Use a prebuilt Arcade to build Arcade -->
     <UseBootstrapArcade>true</UseBootstrapArcade>
+    <!--
+      To ensure that every BuildPass2 vertical has a full Arcade to use without referencing the "rid-agnostic" vertical,
+      set all of Arcade's packages to be internal visibility when we would do RID-specific publishing
+      and disable RID-specific publishing.
+    -->
+    <BuildArgs Condition="'$(EnableDefaultRidSpecificArtifacts)' == 'true'">$(BuildArgs) /p:DefaultArtifactVisibility=Internal</BuildArgs>
```

Comment:
How does this interact with the existing `DefaultArtifactVisibility` settings? https://github.com/dotnet/sdk/blob/e7f6cf2976ffcda43adf8588d298653f1093cbab/src/SourceBuild/content/repo-projects/Directory.Build.props#L116-L119